### ISSUE TITLE
[Feature request] iOS/Samsung like preview (scrollable) bar when viewing images

### ISSUE THREAD

**Issue by bertin0** [deckerst/aves#378]
A nice feature that I've seen originally on iOS and is also copied on Samsung's gallery that I really like and I haven't found on any (Foss) gallery app is the bottom preview bar which has small thumbnails of the photos/videos available and that you can scroll to quickly go through them.

**Comment by deckerst** [comment]
You can try Settings > Viewer > Overlay > Show thumbnails.

However, at the time i implemented that, the performance for switching was not so great so I added some delay between scrolling through the thumbnails and the image appearing. I'd like to revisit this eventually, so that the change is instantaneous.

**Comment by bertin0** [comment]
Thanks, i hadn't noticed that option before. If you're gonna revisit this please also make them bigger. Thanks for the quick response!

**Comment by marcosilva85** [comment]
Such a brilliant feature! I've come across on it on iOS and Samsung, and cannot find a reason not to have it for Aves too!

Is there any reason why it hasn't been enabled yet?

**Comment by deckerst** [comment]
@marcosilva85 have you tried [that](https://github.com/deckerst/aves/issues/378#issuecomment-1296357993)?

**Comment by marcosilva85** [comment]
Yes, but with the artificial delay it's not even close to iOS/Samsung usability:
In iOS I recall I kept my finger over the small thumbnails and would slide across along photos while keeping my finger on the screen, navigating forward and back as I needed. I used it a lot, was my favorite way to navigate across photos.

In Aves, though I have it enabled I barely use it as it does not have this UX.

**Comment by snakedoctr** [comment]
Just wanted to say thanks for an awesome app!  This feature is something I'd use as well.  I tried the suggestion, but it's not responsive enough to be useful, currently.  I'd love to see it implemented/improved upon similar to how iOS' Photos app behaves (with the same responsiveness).  It's something I use all the time when I'm in Photos on an iOS device I use, and would use it just the same if Aves had it implemented similarly.  

Thanks again for a great app!

**Comment by dgudim** [comment]
Bumping this. Can the delay be configured? Maybe on more powerful devices the performance drop you mentioned won't be that noticable.
